shepherd/Goal.py

```diff
@@ -67,4 +78,50 @@ def apply_powerup(self, effect, alliance):
             effect - enum representing an effect to be applied to the goal
             alliance - the alliance trying to apply the effect to the goal
         """
-        pass
+        if effect == POWERUP_TYPES.TWO_X:
+            if alliance.name == CONSTANTS.BLUE:
```

Comment:
I think the `CONSTANTS` name space should be reserve for things with numerical values (e.g. multipliers and things). Perhaps add `CONSTANTS` into a new class.

Reply:
Possibly use an `ENUMS` class for strings an `CONSTANTS` for numerical values